packages/hub/node-tests/services/scheduled-payments/executor-test.ts

```diff
@@ -5,6 +5,7 @@ import ScheduledPaymentsExecutorService from '../../../services/scheduled-paymen
 import { setupStubWorkerClient } from '../../helpers/stub-worker-client';
 import BN from 'bn.js';
 import CrankNonceLock from '../../../services/crank-nonce-lock';
+import { BigNumber } from 'ethers';
```

Comment:
Is adding `BigNumber` needed when we're already using `BN`?

Reply:
reverted this update